### PR TITLE
Jeanregisser/cb earn hotfixes

### PR DESCRIPTION
**Pull request by jeanregisser**
Changes on top of #196:

- add and test Komenci ready endpoint (done successfully on alfajores)
- add timeout (30secs) when calling the Komenci ready endpoint
- add more logs when calling the Komenci ready endpoint
- require phone number to tap "Continue" when in verification education screen (but can still skip)
- copy changes in verification education screen
- fix TS build

### CODE REVIEW

File: packages/mobile/src/geth/networkConfig.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ const CURRENT_MTW_IMPLEMENTATION_ADDRESS_MAINNET: Address =
 const CURRENT_MTW_IMPLEMENTATION_ADDRESS_STAGING: Address =
   '0x5C9a6E3c3E862eD306E2E3348EBC8b8310A99e5A'
 
-const KOMENCI_LOAD_CHECK_ENDPOINT_STAGING = 'PUT_ENDPOINT_HERE'
+const KOMENCI_LOAD_CHECK_ENDPOINT_STAGING = 'https://staging-komenci.azurefd.net/v1/ready'
 const KOMENCI_LOAD_CHECK_ENDPOINT_PROD = 'PUT_ENDPOINT_HERE'
```
Comment (gnardini) [inline packages/mobile/src/geth/networkConfig.ts:72]: Even if it's not live yet the url should be `https://mainnet-komenci.azurefd.net/v1/ready`, right?

**Review by gnardini — COMMENTED** [review]
🚀

**Review by tarikbellamine — APPROVED** [review]
Looks good, thanks!

File: packages/mobile/src/verify/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      const response = (await Promise.race([
+        fetch(networkConfig.komenciLoadCheckEndpoint),
+        delay(FETCH_TIMEOUT, new Error('Timeout Error')),
+      ])) as Response
+      const isReady = response.ok // status in the range 200-299
+      Logger.info(TAG, `Komenci isReady=${isReady} (statusCode=${response.status})`)
+      return isReady
+    } catch (error) {
+      Logger.error(TAG, 'Failed to determine komenci readiness', error)
+      throw error
```
Comment (jmrossy) [inline packages/mobile/src/verify/hooks.ts:26]: So if the network request fails, this error will propagate and the app will go to the ErrorBoundary fail screen right? Or does useAsync swallow errors? Just confirming the desired behavior.

**Review by jmrossy — APPROVED** [review]
LGTM, just one question

File: packages/mobile/src/verify/hooks.ts (hunk 2 above)
Comment (jeanregisser) [reply packages/mobile/src/verify/hooks.ts:26]: `useAsync` will handle it, i.e. swallow it

**Pull request merged**